src/MAKE/MACHINES/Makefile.theta

```diff
@@ -1,7 +1,5 @@
 # knl = Flags for Knights Landing Xeon Phi Processor,Intel Compiler/MPI,MKL FFT
```

Comment:
Should be `theta = ` here instead of `knl =` to match the name of the Makefile when using the "make" command.